### ISSUE TITLE
[Question] Forcing version type

### ISSUE THREAD

**Issue by kylemh** [changesets/changesets#766]
I read all of the extensive documentation, but I see that the CLI has no options for forcing a type of versioning.

For example `pnpm changesets --patch`

I'm trying to provide a simple command for an application like `pnpm release-to-prod` that runs changesets with some of the options answered for. Is there a way to do this?

**Comment by Andarist** [comment]
Nope, this is **currently** not possible - but that would be a good feature. PRs are welcome.

**Comment by bluwy** [comment]
I'm closing as there hasn't been other requests for this and PRs. But `--patch` is now supported as `--patch <pkg-name>` to pre-specify the packages and bump type, which may also be useful here.
